docs/IDE_COMPATIBILITY.md

```
@@ -0,0 +1,446 @@
+
+### Deploy API (Cloud Run)
+
+```bash
+# Deploy to your GCP project
+cd infrastructure/api
+gcloud run deploy versatil-api \
+  --source . \
+  --region us-central1 \
+  --project centering-vine-454613-b3 \
+  --allow-unauthenticated
+```
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)